contribs/gmf/src/directives/backgroundlayerselector.js

```diff
@@ -166,9 +175,37 @@ gmf.BackgroundlayerselectorController.prototype.$onInit = function() {
 gmf.BackgroundlayerselectorController.prototype.handleThemesChange_ = function() {
   this.gmfThemes_.getBgLayers().then((layers) => {
     this.bgLayers = layers;
+
+    if (this.opacityOptions !== undefined && !this.isThemeLoadedOnce_) {
```

Comment:
This method is called several times, it should be normal.
I would avoid messing with the ordering inside `getBgLayers` so I suggest to duplicate the array before changing it (use slice without arguments).
Then remove the `isThemeLoadedOnce` condition.